index.js

```diff
@@ -54,12 +56,13 @@ class CircuitBreaker {
             this.breaker.apply(this.breaker, args)
```

Comment:
Can you add that to the test then, make sure the breaker doesn't count it as a failure?